frontend/src/metabase/home/components/HomeLayout/HomeLayout.styled.tsx

```diff
@@ -40,15 +40,21 @@ export const LayoutBody = styled.div`
   }
 `;
 
-export const LayoutIllustration = styled.div`
+export const LayoutIllustration = styled.div<{
+  backgroundImageSrc: string;
+  isDefault: boolean;
+}>`
   position: absolute;
   top: 0;
   left: 0;
   width: 100%;
   height: 100%;
-  filter: hue-rotate(${hueRotate("brand")}deg);
-  background-image: url("app/img/bridge.svg");
-  background-size: max(min(1728px, 260vh), 100%) auto;
+  filter: ${({ isDefault }) =>
```

Comment:
The change is basically the same as `frontend/src/metabase/auth/components/AuthLayout/AuthLayout.styled.tsx`